src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+
+    protected String color;
```

Comment:
It would be great if you will make type of this variable Color instead of String